reference/UQ Handbook Chapter.ipynb

```diff
@@ -91,12 +91,7 @@
     "alpha, beta = 1, 1\n",
     "\n",
     "# Simulate outcome variable\n",
-    "Y = alpha + X*beta + np.random.randn(size)\n",
-    "\n",
-    "# Define helper function\n",
-    "def print_public(module):\n",
-    "    import pprint\n",
-    "    pprint.pprint([name for name in dir(module) if name[0].istitle()])"
+    "Y = alpha + X*beta + np.random.randn(size)"
```

Comment:
I wonder if we shouldn't call X and Y Xdata and Ydata to make it clearer that they're just arrays.